music21/environment.py

```diff
@@ -432,6 +432,19 @@ def _loadDefaults(self, forcePlatform=None):
             ]:
                 self.__setitem__(name, value)  # use for key checking
 
+    def _checkAccessibility(self, path):
+        '''
+        return True if the path exists, is readable and writable.
+        'path' must be a string or a subclass of pathlib.Path
+        '''
```

Comment:
This is private, so not a big deal, but it's usually easier to annotate the type than to spell it out in English.
```suggestion
    def _checkAccessibility(self, path: Union[str, pathlib.Path]) -> bool:
        '''
        Return True if the path exists, and is readable and writable.
        '''
```